Library/Formula/strider.rb

```diff
@@ -0,0 +1,93 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
+  end
+end
+
+class UniversalNode < Requirement
+  fatal true
+  satisfy { which("node") }
+  def message
+    "node is required. You can install this via nvm or by a straightforward `brew install node`."
+  end
+end
+
+class Strider < Formula
+  url "https://github.com/Strider-CD/strider/archive/v1.6.3.tar.gz"
+  sha1 "6f0fe2b306e2bce395d39b9ce2dc6249bdcf616d"
+  homepage "http://stridercd.com/"
+
+  depends_on UniversalNode
+  depends_on UniversalNpm
+  depends_on "mongodb"
+
+  head "https://github.com/Strider-CD/strider.git", :branch => "develop"
+
+  def install
+    system "npm", "install", "-g"
```

Comment:
Homebrew doesn't support global ` npm ` dependency installs. If ` npm ` has to be used, it should be used locally, not globally.